types/node/v7/tslint.json

```diff
@@ -25,6 +25,7 @@
     "no-string-throw": false,
     "no-unnecessary-qualifier": false,
     "no-var-keyword": false,
+    "no-var": false,
```

Comment:
I've just retested against the latest master and unfortunately it seems to still be needed. I'm getting a lot of linting errors `ERROR:` 1162:9  no-var  Do not use 'var'.` if I don't have this.